pyproject.toml

```
@@ -22,6 +22,15 @@ dependencies = [
 [project.entry-points."kimmdy.reaction_plugins"]
 dimerization = "kimmdy_dimerization.reaction:DimerizationReaction"
 
+[project.optional-dependencies]
```

```suggestion
[dependency-groups]
```

`optional dependencies` are for users to install, `dependency-groups`[dev] will be automatically installed by uv when installing from source/editable. More here [uv docs](https://docs.astral.sh/uv/concepts/projects/dependencies/#development-dependencies)